package.json

```diff
@@ -7,7 +7,9 @@
     "mouji": "./src/index.js"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "tsc",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
keep forever

Reply:
can you add some ci that just runs this :tea:

Reply:
can you?